src/base.css

```diff
@@ -1116,16 +1296,28 @@ a#outdated-note:hover {
 	margin: 0 auto;
 	width: 50%;
 	background: maroon;
+	background: var(--bg-outdated);
 	color: white;
+	color: var(--text-outdated);
 	border-radius: 1em;
 	box-shadow: 0 0 1em red;
+	box-shadow: 0 0 1em var(--shadow-outdated);
 	padding: 2em;
 	text-align: center;
 	z-index: 2;
 }
 
+.outdated-warning a#outdated-note {
+	color: white;
+	color: var(--text-outdated);
+}
+.outdated-warning a#outdated-note:hover {
+	background: transparent;
```

Comment:
and this can probably be folded into the previous rule? We're already hiking specificity.